tools/data_exploration/homogeneity_normality.xml

```diff
@@ -0,0 +1,102 @@
+            '$date'
+            '$species'
+            '$variable'
+            '$output_levene'
+            '$plots'
+        ]]>
+    </command>
+    <inputs>
+        <expand macro="explo_input" />
+        <param name="date" label="Select column containing dates" type="data_column" data_ref="input" />
+        <param name="species" label="Select column containing species" type="data_column" data_ref="input" />
+        <param name="variable" label="Select column containing numerical values (like abundances)" type="data_column" data_ref="input" numerical="True" />
+    </inputs>
+    <outputs>
+        <data name="output_levene" from_work_dir="levene.txt" format="txt" />
+        <collection type="list" name="plots">
+            <discover_datasets pattern="(?P&lt;designation&gt;.+)\.png" visible="false" format="png" />
+        </collection>
+    </outputs>
+    <tests>
+        <test>
+            <param name="input" value="data_frenchbbs.tabular" />
+            <param name="colnames" value="true" />
+            <param name="date" value="2" />
+            <param name="species" value="3" />
+            <param name="variable" value="4" />
+            <output name="output_levene" value="levene.txt" />
+            <output_collection name="plots" type="list" count="2">
+                <element name="Homogeneity" ftype="png">
+                    <assert_contents>
+                        <has_text text="PNG" />
+                    </assert_contents>
+                </element>
+                <element name="Normal_distribution" ftype="png">
+                    <assert_contents>
+                        <has_text text="PNG" />
+                    </assert_contents>
+                </element>
+            </output_collection>
+        </test>
+    </tests>
+    <expand macro="topic" />
+    <help><![CDATA[
+=================================
+Compute Homogeneity and normality
+=================================
+
+**What it does**
+
+This tool computes variances of factors of the species according the dates data:
+
+- Multiple graph with boxplot
+
+This tool computes the dispersion of a numerical value and shows if it is normaly distributed :
+
+- Cumputes 2 graph
+
+**Input description**
+
+A tabular file with observation data. Must at least contain three columns
+which associate date, specie and a factor.
+
+A tabular file with observation data. Must at least contain one column
+whith a numerical value such as the abundance.
+
++----------+-----------+----------------+
+|   date   |   number  |  species.code  |
++==========+===========+================+
+|   year   |     4     |    speciesID   |
++----------+-----------+----------------+
+|  ...     |    ...    |       ...      |
++----------+-----------+----------------+
+
+**Output**
+
+A  png file with multiple graph with a for each year boxplot.
+
+A png file with two graphics both showing the normality of the distribution but differently such as the image following.
```

Comment:
No tests are performed to quantify if the distribution is normal ? Like a Shapiro-Wilk test ?